policies/terms_of_membership.md

```diff
@@ -0,0 +1,113 @@
+# Terms of Membership
+
+Last Updated: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+Next Review: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+Designated Membership Liaison: Artemiy Kondratiev (art@farsetlabs.org.uk)
+
+## Charitable Purpose
+
+The Charity’s objects (“Objects”) are the education of the public in STEAM (Science, Technology, Engineering, Arts and Mathematics) and the provision of community accessible workspaces to facilitate STEAM outreach and education in Northern Ireland, primarily focused in the Belfast area
+
+See [here](articles_of_association.md) for more information.
+
+## Membership 
+
+Membership of Farset Labs gives you one vote in matters arising at any annual or extraordinary general meeting. This includes changes to the constitution, appointing or removing the charity trustees, and deciding whether the charity should be wound up. 
+
+As a member you agree to adhere to these Terms of Membership and a serious breach of these terms will result in the termination of membership.
+
+As a member you are granted permission to avail (where other considerations / inductions permit) the use of the space and facilities at our Unit 1 Weavers Court Business Park location. 
+
+Termination of membership will result in withdrawal of permission to use or access the building. 
+
+### Membership Dues
+
+The membership due covers all costs associated with running the building, including: rent, electricity, heating, water, internet connectivity, public liability insurance (for common areas), legal fees, administration costs and general maintenance. There will be a review of membership fees annually to ensure that we are covering the costs of running the building.
+
+Unless otherwise agreed by members, no members will receive a fee or discount to dues/desk rends for carrying out tasks related to the running of the building or charity.
```

Comment:
Typo "rends"

How flexible is this policy? This could potentially be revisited in the future if we change to give discounts etc for volunteering